tests/fixtures/backends.py

```diff
@@ -241,6 +245,67 @@ def anyio_backend():
     return "asyncio"
 
 
+@pytest.mark.anyio
+@pytest.fixture(params=["sync", "async"])
+def lrs_backend(
+    request,
```

Comment:
```suggestion
    flavor,
```
`request` is a bit confusing terminology imo

Reply:
I was fine with the `request` terminology, as I already parametrized pytest fixture in the past.
But I'm OK with adding an alias for clarity!